deploy/examples/create-external-cluster-resources.py

```diff
@@ -826,7 +944,46 @@ def validate_rados_namespace(self):
             raise ExecutionFailureException(
                 ("The provided rados Namespace, '{}', is not found in the pool '{}'").format(
                     rados_namespace, rbd_pool_name))
-
+    
+    def get_rgw_fsid(self):
+        access_key = self.out_map['RGW_ADMIN_OPS_USER_ACCESS_KEY']
+        secret_key = self.out_map['RGW_ADMIN_OPS_USER_SECRET_KEY']
+        rgw_endpoint = self._arg_parser.rgw_endpoint
+        cert = None
+        verify = None
+        if self._arg_parser.rgw_tls_cert_path and not self._arg_parser.rgw_skip_tls:
+            cert = self.validate_rgw_endpoint_tls_cert()
+            verify = True
+        if self._arg_parser.rgw_skip_tls:
+            verify = False
+        base_url =  self.endpoint_dial(rgw_endpoint,cert=cert) + "://"
+        base_url = base_url + rgw_endpoint + "/admin/info?"   
+        params = {'format': 'json'}
+        request_url = base_url + urllib.parse.urlencode(params)
+            
+        try:
+            r = requests.get(request_url, auth=S3Auth(access_key, secret_key,rgw_endpoint), cert=cert, verify=verify)
+        except requests.exceptions.Timeout:
+            raise ExecutionFailureException(
+                    "invalid endpoint:, not able to call admin-ops api{}".format(rgw_endpoint))
+        r1 = r.json()
+        if r1 is None or r1.get('info') is None:
+            return "NULL"
```

Comment:
Let's return an exception for this case? or just an empty string?

Reply:
Returned empty string, as an exception will be thrown by validate_rgw_endpoint